tests/mocks.py

```
@@ -0,0 +1,432 @@
+sys.modules["requests"] = MagicMock()
+
+
+class RequestException(Exception):
+    pass
+
+
+class HTTPError(RequestException):
+    pass
+
+
+class ConnectionError(
```

Ruff (0.15.12)

[error] Variable `ConnectionError` is shadowing a Python builtin

(A001)